### DISCUSSION TITLE
SyntaxError: The requested module 'node:events' does not provide an export named 'addAbortListener'

### DISCUSSION THREAD

**Discussion by wu529778790** [vuepress/core#1597] [Q&A] [answered]
import {addAbortListener} from 'node:events';
        ^^^^^^^^^^^^^^^^
SyntaxError: The requested module 'node:events' does not provide an export named 'addAbortListener'
    at ModuleJob._instantiate (node:internal/modules/esm/module_job:124:21)
    at async ModuleJob.run (node:internal/modules/esm/module_job:190:5)

**Comment by Mister-Hope** [comment, marked as answer]
Likely you are using an outdated node version

**Reply by wu529778790** [reply]
You're right, I'll just upgrade to v20.16.0. But the official website documentation requires Node.js v18.16.0+, which is sufficient. I was originally v18.17.0. It seems that the official website has not been updated.

**Reply by KianYang-Lee** [reply]
Faced the same issue and this confirms it. THanks
